packages/manager/apps/web/client/app/email-domain/upgrade/upgrade.controller.js

```diff
@@ -0,0 +1,93 @@
+import head from 'lodash/head';
+import get from 'lodash/get';
+
+export default class EmailDomainUpgradeCtrl {
+  /* @ngInject */
+  constructor(
+    $translate,
+    $window,
+    Alerter,
+    EmailDomainService,
+    MANAGER_URLS,
+  ) {
+    this.$translate = $translate;
+    this.$window = $window;
+    this.Alerter = Alerter;
+    this.EmailDomainService = EmailDomainService;
+    this.MANAGER_URLS = MANAGER_URLS;
```

Comment:
Could those be extracted and scoped to the component ? That way this component comes with the URLs it needs